[2025-10-18T04:22:30.651+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: agricultural_sri_annual_report.report_generation.generate_visualizations manual__2025-10-18T02:31:24.144743+00:00 [queued]>
[2025-10-18T04:22:30.656+0000] {taskinstance.py:1159} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: agricultural_sri_annual_report.report_generation.generate_visualizations manual__2025-10-18T02:31:24.144743+00:00 [queued]>
[2025-10-18T04:22:30.657+0000] {taskinstance.py:1361} INFO - Starting attempt 1 of 3
[2025-10-18T04:22:30.664+0000] {taskinstance.py:1382} INFO - Executing <Task(PythonOperator): report_generation.generate_visualizations> on 2025-10-18 02:31:24.144743+00:00
[2025-10-18T04:22:30.666+0000] {standard_task_runner.py:57} INFO - Started process 988 to run task
[2025-10-18T04:22:30.669+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'agricultural_sri_annual_report', 'report_generation.generate_visualizations', 'manual__2025-10-18T02:31:24.144743+00:00', '--job-id', '34', '--raw', '--subdir', 'DAGS_FOLDER/agricultural_sri_pipeline.py', '--cfg-path', '/tmp/tmpwajvql32']
[2025-10-18T04:22:30.669+0000] {standard_task_runner.py:85} INFO - Job 34: Subtask report_generation.generate_visualizations
[2025-10-18T04:22:30.694+0000] {task_command.py:416} INFO - Running <TaskInstance: agricultural_sri_annual_report.report_generation.generate_visualizations manual__2025-10-18T02:31:24.144743+00:00 [running]> on host 4e2b74bc69c8
[2025-10-18T04:22:30.728+0000] {taskinstance.py:1662} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='alerts@agcompany.com' AIRFLOW_CTX_DAG_OWNER='agricultural_risk_team' AIRFLOW_CTX_DAG_ID='agricultural_sri_annual_report' AIRFLOW_CTX_TASK_ID='report_generation.generate_visualizations' AIRFLOW_CTX_EXECUTION_DATE='2025-10-18T02:31:24.144743+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-18T02:31:24.144743+00:00'
[2025-10-18T04:22:31.097+0000] {font_manager.py:1639} INFO - generated new fontManager
[2025-10-18T04:22:31.400+0000] {warnings.py:109} WARNING - /home/***/.local/lib/python3.10/site-packages/***/utils/context.py:314: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2025-10-18T04:22:31.401+0000] {agricultural_sri_pipeline.py:497} INFO - ðŸ“ˆ Generating visualizations for 2025
[2025-10-18T04:22:31.405+0000] {taskinstance.py:1937} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 192, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 209, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/agricultural_sri_pipeline.py", line 502, in generate_visualizations
    result = generate_all_visualizations(
TypeError: generate_all_visualizations() got an unexpected keyword argument 'sri_path'
[2025-10-18T04:22:31.408+0000] {taskinstance.py:1400} INFO - Marking task as UP_FOR_RETRY. dag_id=agricultural_sri_annual_report, task_id=report_generation.generate_visualizations, execution_date=20251018T023124, start_date=20251018T042230, end_date=20251018T042231
[2025-10-18T04:22:31.412+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 34 for task report_generation.generate_visualizations (generate_all_visualizations() got an unexpected keyword argument 'sri_path'; 988)
[2025-10-18T04:22:31.436+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 1
[2025-10-18T04:22:31.446+0000] {taskinstance.py:2778} INFO - 0 downstream tasks scheduled from follow-on schedule check
